README.md

```diff
@@ -28,16 +28,16 @@ Follow one of the following ways to add the JAR in the file:
 
 * Download the JAR and update the path.
     ```
-    [[platform.java17.dependency]]
+    [[platform.java21.dependency]]
     path = "PATH"
     ```
 
 * Add JAR with the maven dependency params.
     ```
-    [[platform.java17.dependency]]
+    [[platform.java21.dependency]]
     groupId = "com.microsoft.sqlserver"
     artifactId = "mssql-jdbc"
-    version = "10.2.0.jre17"
+    version = "10.2.0.jre21"
```

Comment:
i think we need to use java 21 support version. seems we need 12.6

Reply:
I'll update